src/afl-fuzz-init.c

```diff
@@ -1078,38 +1085,94 @@ void perform_dry_run(afl_state_t *afl) {
 
         }
 
-        q->disabled = 1;
-        q->perf_score = 0;
+        /* Crashing corpus will regrad as normal, and categorized as new crash at fuzzing  */
```

Comment:
typos: `Crashing seeds will be regarded as new crashes on startup` or something like that